dashboard/src/actions/charts.ts

```diff
@@ -149,36 +141,43 @@ export function getChartReadme(
   };
 }
 
-export function getChartValues(
+export function getChartVersionWithValuesAndSchema(
   id: string,
   version: string,
 ): ThunkAction<Promise<void>, IStoreState, null, ChartsAction> {
   return async dispatch => {
     try {
-      const values = await Chart.getValues(id, version);
-      dispatch(selectValues(values));
-    } catch (e) {
-      dispatch(selectValues(""));
-    }
-  };
-}
-
-export function getChartSchema(
-  id: string,
-  version: string,
-): ThunkAction<Promise<void>, IStoreState, null, ChartsAction> {
-  return async dispatch => {
-    try {
-      const schema = await Chart.getSchema(id, version);
-      dispatch(selectSchema(schema));
-    } catch (e) {
-      if (e.constructor === NotFoundError) {
-        // Schema not found
-        dispatch(selectSchema({}));
-      } else {
-        // Unexpecter error
-        dispatch(errorChart(e));
+      dispatch(requestCharts());
+      const chartVersion = await Chart.getChartVersion(id, version);
+      if (chartVersion) {
+        let values = "";
+        let schema = {};
+        try {
```

Comment:
This try and the one below it look they could be DRY'd up a bit (the code is identical for both other than the function name?)